docs/en/guides/asf/committer.md

```diff
@@ -112,14 +109,17 @@ At this point, congratulate! You have the official Apache ID.
 The PMC member should add the new committer to official committer list through [roster](https://whimsy.apache.org/roster/committee/skywalking). 
 
 ### Set up the Apache ID and dev env
-1. Go to [Apache Account Utility Platform](https://id.apache.org/), initial your password, set up your personal mailbox(`Forwarding email address`) 
-and GitHub account(`Your GitHub Username`).
-1. If you want to use `xxx@apache.org` to send mail, you need to use GMail, and following [this guide](https://reference.apache.org/committer/email).
-1. Use [GitBox Account Linking Utility](https://gitbox.apache.org/setup/) to make your GitHub account to join Apache GitHub org.
-Notice, if never got your GitHub invitation? Visit github.com/apache to see if you have an invitation pending. Also, your GitHub MFA Status is required.
-
-If you want others could see you are in the Apache GitHub org, you need to go to [Apache GitHub org people page](https://github.com/orgs/apache/people), 
-search for yourself, and choose `Organization visibility` to `Public`.
+1. Go to [Apache Account Utility Platform](https://id.apache.org/), initial your password, set up your personal mailbox(`Forwarding email address`) and GitHub account(`Your GitHub Username`). An organisational invite will be sent to you via email shortly thereafter (within 2 hours).
+2. If you want to use `xxx@apache.org` to send mail, please refer to [here](https://infra.apache.org/committer-email.html). Gmail is recommended, because in other mailbox service settings, this forwarding mode is not easy to find.
```

Comment:
You don't need to increase the number yourself, in case other people add one line in the middle of the list and have to change all the numbers (most of us don't use a dedicated software to edit markdown files like you)

Reply:
done